demosplan/DemosPlanCoreBundle/Entity/Procedure/ProcedurePerson.php

```diff
@@ -136,9 +136,19 @@ public function setFullName(?string $fullName): ProcedurePerson
         return $this;
     }
 
-    public function getStreetName(): ?string
+    public function getStreetName(bool $withStreetNumber = false): ?string
```

Comment:
Perhaps you should better write a new function `getStreetNameWithNumber()`

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/2069/commits/47223523a2495db997954d7cbd8760d2a5bccbfd